### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Epse**
I added Dutch translation files, as per issue #3

### CODE REVIEW

File: lang/nl.po [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_list_badargument"
+msgstr "%s: het argument voor de optie --list moet 'none', 'free', 'nonfree' of 'all' zijn."
+msgid "msg_unknown_option"
+msgstr "%s: onbekende optie '%s'"
+msgid "msg_total_free"
+msgstr "%d vrije paketten"
+msgid "msg_total_nonfree"
+msgstr "%d niet-vrije paketten"
+msgid "msg_rms_happy_ascii"
+msgstr "Enkel vrije paketten - rms zou trots zijn!                           rms is blij."
```
Comment (suve) [inline lang/nl.po]: This text should be exactly 79 characters long so it fits nicely on a standard terminal width.

File: lang/nl.po [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_rms_happy"
+msgstr "Enkel vrije paketten - rms zou trots zijn!"
+msgid "msg_rms_disappoint_ascii"
+msgstr "Meer dan 10%% niet-vrije paketten. Haat jij vrijheid?              rms is teleurgesteld."
```
Comment (suve) [inline lang/nl.po]: Like above, should not exceed 79 characters.

File: man/nl.man [hunk 3]
```diff
@@ -0,0 +1,62 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - maakt een rapport van geïnstalleerde niet-vrije software
+.SH Samenvatting
+\fBvrms-rpm\fR \fI[OPTIE]\fR ...
+.SH Beschrijving
```
Comment (suve) [inline man/nl.man]: The headers (`.SH`) should be all upper-case.

File: lang/nl.po (hunk 2 above)
Comment (Epse) [reply lang/nl.po]: oh, `msg_list_badargument` seems to be too long in English as well, but I'll fix it

**Comment by Epse** [comment]
updated my branch, have a look!

**Comment by suve** [comment]
Sorry for taking so long. Everything looks cool now. Merged. Thanks!

**Comment by suve** [comment]
Sorry for taking so long. Everything looks cool now. Merged. Thanks!

**Pull request merged**